roles/ucs/defaults/main.yml

```diff
@@ -9,24 +9,27 @@ operator_group: "{{ operator_user }}"
 # docker
 
 docker_registry: index.docker.io
-docker_registry_ucs: "{{ docker_registry }}"
+docker_registry_ucs: docker-test.software-univention.de
 
 ##########################
 # ucs
 
-ucs_hostname: ucs-master
-ucs_domainname: osism.local
-ucs_root_password: password
-
-ucs_configuration_directory: /opt/ucs/configuration
-ucs_docker_compose_directory: /opt/ucs
+ucs_configuration_directory: /opt/ucs-iam/configuration
```

Comment:
Directory name == role name. Please do not change.

Reply:
Ok, thanks, I'll fix that and the corresponding points below.